driver/product/compile_gfsv152.sh

```diff
@@ -0,0 +1,20 @@
+
+
+set -x
+
+export version=v15.2.0
+
+cd /gpfs/dell2/emc/modeling/noscrub/Boi.Vuong/git/gfs.$version/sorc
```

Comment:
Is this script needed?  It's pointing to personal paths in several places.

Reply:
There are used by developers to test jobs outside of the workflow.  It is better to keep them. There is no need to clean them up either.

Reply:
how does a developer `cd` into Boi.Vuong's directory and write inside it?

Reply:
Rahul, all scripts under ./driver are only used by individual developers to test the jobs who are responsible for.  It will take considerable effort to clean them up to follow the "standard".  The entire ./driver directory can either be removed from the workflow or kept there for easy access for the next round of development.

Reply:
I will leave this script for now and open an issue to look at removing or updating the driver scripts for upcoming development.